data/remnant missions.txt

```diff
@@ -1252,3 +1252,237 @@ mission "Remnant: Expanded Horizons Quarg 3"
 			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he says while tapping the databanks, "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
 			`	Upon arriving at <planet> they quickly thank you, hand you <payment>, and run back to their offices to finish prepping their material to publish and distribute.`
 
+
+
+mission "Remnant: Broken Jump Drive 1"
+	name "Broken Jump Drive Delivery"
+	description "You have been asked to deliver a broken jump drive to the researchers on <planet>."
+	minor
+	landing
+	to offer
+		has "remnant blood test pure"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`As the <ship> settles onto its designated landing pad you see someone emerge from the nearby outfitter and quickly make their way over to your ship. They wait a safe distance back, then approaches once you cycle the hatch and disembark. "Greetings, Captain <last>," trills the Remnant. "We saw the news that the newcomer among us was tracking down the erratic Korath ship. When you came in to land we saw on the scans that you managed to salvage its malfunctioning jump drive. Could we take it to study?"`
```

Comment:
- possibly "As your ship the `<ship>`"
 - possibly needed comma between "pad" and "you"
 - "they wait" is plural but "approaches" is singular.
 - possibly "make their way to you" instead of "make their way over to your ship"
 - "land, we"
 - "managed to salvage" -> "salvaged"
 - "it, to study?"

Possibly the player is still tracking down other erratic Korath ships, so that could be present tense. (You could `branch` based on active bounty missions > 0, since any that are redeemed on this planet will already be completed before this can offer).

Reply:
Changed all of them. Thanks!

Also changed it from `tracking down the erratic Korath` to `tracking down an erratic Korath`. 

The news itself that the player is tracking down the Korath would be in the past regardless, but yes, a branch could be done here. There is also the possibility that the player has disabled the Korath ship (and thus is no longer actually tracking it), but hasn't disabled it yet (and thus still has the mission active).

Reply:
Ref: `As your ship, the <ship>,` Seems clunky. We *did* do that poll on Discord where most people said they know what their flagship name is, so I don't think it makes sense to remind people what their ship name is like this.

Reply:
I like the sounds of it better without the clunkiness. Changed.